L.SimpleGraticule.js

```diff
@@ -116,15 +117,15 @@ L.SimpleGraticule = L.LayerGroup.extend({
     },
 
     buildXLine: function(x) {
-        var bottomLL = new L.LatLng(this._bounds.getSouth(), x);
-        var topLL = new L.LatLng(this._bounds.getNorth(), x);
+        var bottomLL = new L.LatLng(-180, x);
+        var topLL = new L.LatLng(180, x);
 
         return new L.Polyline([bottomLL, topLL], this.lineStyle);
     },
 
     buildYLine: function(y) {
-        var leftLL = new L.LatLng(y, this._bounds.getWest());
-        var rightLL = new L.LatLng(y, this._bounds.getEast());
+        var leftLL = new L.LatLng(y, -360);
```

Comment:
I'm not sure I follow the rationale behind these values.  The point of SimpleGraticule is to be used in L.CRS.Simple: an infinite Cartesian plane.  You can have values well below or above 180, 360